test.ini

```diff
@@ -9,7 +9,7 @@ host = 0.0.0.0
 port = 5000
 
 [app:main]
-use = config:../ckan/test-core.ini
+use = config:/usr/local/lib/ckan/default/src/ckan/test-core.ini
```

Comment:
I'm not certain that everyone will be installing in this exact folder. Any reason we are going absolute instead of relative?